<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1273</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 110, 110);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="m_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>341</width>
      <height>921</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="la_br">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_brightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::handle:horizontal {
    border: 10px solid rgb(71, 71, 71);
}

QSlider::handle:horizontal:hover {
    border: 10px solid rgb(122, 121, 121);
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="la_con">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_contrast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::handle:horizontal {
    border: 10px solid rgb(71, 71, 71);
}

QSlider::handle:horizontal:hover {
    border: 10px solid rgb(122, 121, 121);
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="la_sh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Sharpness</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_sharpness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::handle:horizontal {
    border: 10px solid rgb(71, 71, 71);
}

QSlider::handle:horizontal:hover {
    border: 10px solid rgb(122, 121, 121);
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="la_sat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
    <widget class="QSlider" name="sl_saturation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>311</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::handle:horizontal {
    border: 10px solid rgb(71, 71, 71);
}

QSlider::handle:horizontal:hover {
    border: 10px solid rgb(122, 121, 121);
}
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_negative">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Negative</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_blackwhite">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>590</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>BlackWhite</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_shadesofgray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Shades of gray</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sepia">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>590</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Sepia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_noise">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>650</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Noise</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sky">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>650</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Sky</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_gold">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>710</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Gold</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_twilight">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>710</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Twilight</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sweetpeach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>710</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Sweet Peach</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(97, 97, 97);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="la_cor">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>Correction</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="la_cor_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>Filters</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(97, 97, 97);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="la_cor_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>311</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 19pt &quot;Tempus Sans ITC&quot;;</string>
       </property>
       <property name="text">
        <string>Filters</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_br">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>240</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_con">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sh">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sat">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>420</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>50%</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>770</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(97, 97, 97);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="la_cor_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 19pt &quot;Tempus Sans ITC&quot;;</string>
      </property>
      <property name="text">
       <string>Look</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_rotate">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>860</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Rotate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_mirror">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>860</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Mirror</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_shrunk">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>860</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Shrunk</string>
     </property>
    </widget>
    <zorder>la_br</zorder>
    <zorder>frame</zorder>
    <zorder>sl_brightness</zorder>
    <zorder>la_con</zorder>
    <zorder>sl_contrast</zorder>
    <zorder>la_sh</zorder>
    <zorder>sl_sharpness</zorder>
    <zorder>la_sat</zorder>
    <zorder>sl_saturation</zorder>
    <zorder>btn_negative</zorder>
    <zorder>btn_blackwhite</zorder>
    <zorder>btn_shadesofgray</zorder>
    <zorder>btn_sepia</zorder>
    <zorder>btn_noise</zorder>
    <zorder>btn_sky</zorder>
    <zorder>btn_gold</zorder>
    <zorder>btn_twilight</zorder>
    <zorder>btn_sweetpeach</zorder>
    <zorder>frame_3</zorder>
    <zorder>label_br</zorder>
    <zorder>label_2</zorder>
    <zorder>label_con</zorder>
    <zorder>label_sh</zorder>
    <zorder>label_sat</zorder>
    <zorder>frame_6</zorder>
    <zorder>btn_rotate</zorder>
    <zorder>btn_mirror</zorder>
    <zorder>btn_shrunk</zorder>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 97, 97);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="w2_save">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="w2_reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="w2_open">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: 
	12pt &quot;Tempus Sans ITC&quot;;
	background-color: rgb(95, 95, 95);
}

QPushButton:hover {
	background-color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>861</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(110, 110, 110);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>59</y>
      <width>931</width>
      <height>841</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>861</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>frame_4</zorder>
   <zorder>m_left</zorder>
   <zorder>frame_2</zorder>
   <zorder>image</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1273</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
